packages/react/src/components/TextArea/TextArea.js

```diff
@@ -102,7 +119,10 @@ const TextArea = React.forwardRef(function TextArea(
           ? classNames(`${prefix}--form-item`, className)
           : `${prefix}--form-item`
       }>
-      {label}
+      <div className={`${prefix}--text-area__counter-wrapper`}>
+        {label}
+        {counter}
+      </div>
```

Comment:
I think we may want to do something like 

```suggestion
{label}
      {enableCounter && (
        <div className={`${prefix}--text-area__counter-wrapper`}>{counter}</div>
      )}
```

so that the `text-area__counter-wrapper` class is only applied when the `enableCounter` prop is provided.

Reply:
make sense!

Reply:
Well with the latest update, i wanted to wrap the label and the counter in a div to use flex box to position them relative to each other, i think doing this would break that but i would need to test, what are your thoughts on that? wrapping both and using flexbox to position it or wrapping the element individually and finding some other styling to making it huig the right side? @tw15egan

Reply:
Ah okay, I thought they were going to stack. My mistake. Yes, we will need to have a wrapper around it then. Since it will always be there, perhaps we should just name it `text-area__label-wrapper` so it makes sense even if the counter is not present.

Then you can just do a `display: flex` and `justify-content: space-between` to the wrapper to space them.

Reply:
cool, will change!